### PR TITLE
Documentation: Corrections to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by davidlunarg**
Change-Id: I5f250d11baa29527fd05f0dfdcb267559aacafc3

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,36 +1,31 @@
-## How to Contribute to Vulkan Source Repositories
+## How to Contribute to the VulkanTools Repository
 
 ### **The Repository**
 
-The source code for The Vulkan-ValidationLayer components is sponsored by Khronos and LunarG.
-* [Khronos Vulkan-ValidationLayers](https://github.com/KhronosGroup/Vulkan-ValidationLayers)
+The source code for the VulkanTools components is sponsored by Khronos and LunarG.
```
Comment (lunarpapillo) [inline CONTRIBUTING.md:5]: I didn't think Khronos was sponsoring **VulkanTools**?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -86,53 +81,21 @@ that to be accepted into the repository, the pull request must [pass all tests](
...
 ### **Contributor License Agreement (CLA)**
 
 You will be prompted with a one-time "click-through" CLA dialog as part of submitting your pull request
 or other contribution to GitHub.
 
 ### **License and Copyrights**
 
-All contributions made to the Vulkan-ValidationLayers repository are Khronos branded and as such,
+All contributions made to the VulkanTools are LunarG branded and as such,
 any new files need to have the Khronos license (Apache 2.0 style) and copyright included.
```
Comment (lunarpapillo) [inline CONTRIBUTING.md:99]: "Khronos license" => "LunarG license", or not?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,36 +1,31 @@
...
 ### **The Vulkan Ecosystem Needs Your Help**
 
-The Vulkan validation layers are one of the larger and more important components in this repository.
-While there are often active and organized development efforts underway to improve their coverage,
-there are always opportunities for anyone to help by contributing additional validation layer checks
-and tests for these validation checks.
+The tools in VulkanTools are important components in the Vulkan Ecosystem.  While there are often
+active and organized development efforts underway to improve these tools, there are always
+opportunities for anyone to help by contributing new tools or additional features to existing tools.
+these validation checks.
```
Comment (charles-lunarg) [inline CONTRIBUTING.md:14]: I think this sentence was supposed to be deleted, as there is a period on the previous line...

Comment (KarenGhavam-lunarG) [reply CONTRIBUTING.md:14]: > ### **The Vulkan Ecosystem Needs Your Help**
 I guess I would like the title to be more like: Contributions welcomed. The Vulkan ecosystem will benefit

File: CONTRIBUTING.md (hunk 2 above)
Comment (KarenGhavam-lunarG) [reply CONTRIBUTING.md:99]: It's not a lunarG license either. I would simply say any new files need to have the Apache 2.0 license.

**Pull request merged**